src/scss/colors.scss

```diff
@@ -0,0 +1,25 @@
+//Main colors
+$game-background: #eee8ea;
+$accent-color: #591D2D;
```

Comment:
you can skip the `color` part from color variable names - it will be implied through the imports ;)